lib/license_scout/overrides.rb

```diff
@@ -402,9 +402,11 @@ def default_overrides
         ["slack-notifier", "MIT", ["https://raw.githubusercontent.com/stevenosloan/slack-notifier/master/LICENSE"]],
         ["sslshake", "MPL-2.0", ["https://raw.githubusercontent.com/arlimus/sslshake/master/README.md"]],
         ["sprockets", "MIT", ["https://raw.githubusercontent.com/rails/sprockets/master/MIT-LICENSE"]],
+        ["sqlite3-ruby", nil,["https://raw.githubusercontent.com/sparklemotion/sqlite3-ruby/master/LICENSE"]],
```

Comment:
looks like this is a BSD-3 license https://raw.githubusercontent.com/sparklemotion/sqlite3-ruby/master/LICENSE and https://rubygems.org/gems/sqlite3/versions/1.3.11